common/exception/src/exception.rs

```diff
@@ -189,6 +189,7 @@ build_exceptions! {
     UnknownColumn(58),
     InvalidSourceFormat(59),
     StrParseError(60),
+    IllegalGrant(1144),
```

Comment:
It would be nice order by Code(1144) number.

Reply:
do we try keep the error code compatibile with mysql or not?

if not, i think we can append it after `StrParseError(60)` as `IllegalGrant(61)`. therre's a "// uncategorized" category after the 600 code, but this error is a trivial sql error like `UnknownTable` I think